### ISSUE TITLE
Test/fix Forwarded Authentication (PREAUTHHEADER)

### ISSUE THREAD

**Issue by proycon** [proycon/clam#55]
Possible bug in forwarded authentication; INT reports receiving HTTP 401 despite seemingly correct configuration.

**Comment by proycon** [comment]
(delaying this to next week because of other priorities)

**Comment by tancque** [comment]
Some extra information concerning the setup and debugdata:

- The clamservice in question is piccl, running from lamachine in Centos 7, started as developmentserver.
- Authentication is provided by shibboleth server  proxying the request to the lamachine server via apache webserver.
-  Provided in files are the sniffed headers on the clamserver, the config file of the clamserver and the versionsfile of lamachine. 

 In our first test we used AJP_eppn as ident header but we also tried REMOTE_USER to no avail. Debug responses with 1 line citing http ressponse code 401


[headers.txt](https://github.com/proycon/clam/files/1414037/headers.txt)
[VERSION.txt](https://github.com/proycon/clam/files/1414039/VERSION.txt)
[picclservice.py.txt](https://github.com/proycon/clam/files/1414056/picclservice.py.txt)
